tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -918,6 +926,7 @@ func init() {
     actionInvokeCmd.Flags().BoolVarP(&flags.action.result, "result", "r", false, wski18n.T("blocking invoke; show only activation result (unless there is a failure)"))
 
     actionGetCmd.Flags().BoolVarP(&flags.common.summary, "summary", "s", false, wski18n.T("summarize action details"))
+    actionGetCmd.Flags().BoolVarP(&flags.action.url, "url", "r", false, wski18n.T("get action url"))
```

Comment:
How does the CLI do when both the `--url` flag and `--summary` flag are specified.  Is one ignored (if so, how does user know in doc and in the results)?  Both honored?

And if both `--url` and the `[FIELD_FILTER]` are specified together?